types/v56/types.go

```diff
@@ -2661,8 +2661,8 @@ type Disk struct {
 	Description     string           `xml:"Description,omitempty"`
 	Files           *FilesList       `xml:"Files,omitempty"`
 	Link            []*Link          `xml:"Link,omitempty"`
-	Owner           *Owner           `xml:"Owner,omitempty"`
 	StorageProfile  *Reference       `xml:"StorageProfile,omitempty"`
+	Owner           *Owner           `xml:"Owner,omitempty"`
```

Comment:
Any reason for this field placement change?

Reply:
XML has to be in precise order otherwise serialization fails. Placement change - is the fix